server/api/crud/model_monitoring/deployment.py

```diff
@@ -891,10 +907,19 @@ async def delete_monitoring_function(
                     logger.debug(f"{function_name} pod found, retrying")
                     time.sleep(5)
 
-            v3io_client = v3io.dataplane.Client(endpoint=mlrun.mlconf.v3io_api)
-            stream_paths = server.api.crud.model_monitoring.get_stream_path(
-                project=project, function_name=function_name
+            stream_paths.extend(
+                server.api.crud.model_monitoring.get_stream_path(
+                    project=project, function_name=function_name
+                )
             )
+
+        if stream_paths[0].startswith("v3io"):
```

Comment:
It's not trivial to me why there's at least one item in `stream_paths`.
Can't we reach this point in the project deletion process without any stream?

Moreover, can't we have some edge cases in which there are both V3IO and Kafka streams? E.g. after an upgrade and setting new Kafka credentials.

Reply:
In general it's not the case at the moment but to avoid possible future conflicts I'll check if stream_paths exist. As for your second question right now we don't support multiple stream services under the same project.